### ISSUE TITLE
create label* functions for pagure backend

### ISSUE THREAD

**Issue by dustymabe** [packit-service/ogr#147]
Would be nice to be able to manipulate labels on pagure issues. i.e. `get_issue_labels()` `add_issue_labels()`, etc.. 

*update by @lachmanfrantisek:*

TODO: 
- [x] `labels` property for `PagureIssue` class (https://github.com/packit-service/ogr/pull/321 by @cverna )
- [x] `labels` argument for `PagureIssue.create` and `PagureProject.create_issue` (https://github.com/packit-service/ogr/pull/321 by @cverna)
- [ ] `labels` property for `PagurePullRequest` class
- [ ] `labels` argument for `PagurePullRequest.create` and `PagureProject.create_pr`
- [ ] `add_label` methods blocked by the PagureAPI (https://pagure.io/pagure/issue/2175)
- [ ] add tests for all of those

**Comment by lachmanfrantisek** [comment]
Thanks for the request.

We need to look into the [pagure API](https://src.fedoraproject.org/api/0/) to see if it is possible to manipulate with that via API.

**Comment by rpitonak** [comment]
cc @marusinm as I think he already tried to solve this.

**Comment by lachmanfrantisek** [comment]
> I think he already tried to solve this.

Yes, we were discussing that with Marek and there is no API for manipulation with labels from what we know.

**Comment by TomasTomecek** [comment]
Would be nice if someone opened a request at pagure upstream and linked the issue here :)

**Comment by lachmanfrantisek** [comment]
> Would be nice if someone opened a request at pagure upstream and linked the issue here :)

Any volunteer? OK, I'll create an issue. (I have a list of issues for pagure that I would like to create...;-)

**Comment by lachmanfrantisek** [comment]
There is already an issue for that:

https://pagure.io/pagure/issue/2175

**Comment by dustymabe** [comment]
thanks @lachmanfrantisek for linking the two issues

**Comment by lachmanfrantisek** [comment]
We are still blocked by the Pagure API.

**Comment by lachmanfrantisek** [comment]
> We are still blocked by the Pagure API.

Looks like it's not true anymore... Nice!

**Comment by lachmanfrantisek** [comment]
I've updated the description and add the TODO-list.

**Comment by lachmanfrantisek** [comment]
Some parts are still missing.
